actioncable/lib/action_cable/subscription_adapter/compressor.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module ActionCable
+  module SubscriptionAdapter
+    class Compressor
+      def initialize(threshold: 1.kilobyte, compressor: ActiveSupport::Gzip)
+        @threshold = threshold
+        @compressor = compressor
+      end
+
+      def compress(data)
+        return "0#{data}" if data.size < @threshold
+
+        "1#{@compressor.compress(data)}"
```

Comment:
Use proper conditionals:

```suggestion
        if data.size < @threshold
          "0#{data}" 
        else
          "1#{@compressor.compress(data)}"
        end
```